peachjam/js/components/DocumentContent/index.ts

```diff
@@ -7,11 +7,12 @@ import { createTocController, generateHtmlTocItems, wrapTocItems } from '../../u
 import EnrichmentsManager from './enrichments-manager';
 import i18next from 'i18next';
 
+const bootstrap = (window as { [key: string]: any }).bootstrap;
```

Comment:
When does the boostrap JS get imported? Because this line will be run the moment it's encountered in the JS, so the bootstrap code must be run before it -- do we know that will always happen?

Reply:
So, bootstrap sometimes isn't loaded before the previous line `(window as { [key: string]: any }).bootstrap.Tab(tocTabTriggerEl)` runs. That's why Bootstrap returns undefined. This just prevents the Tab line from running before Bootstrap loads.

But, ideally, Bootstrap should load before that line (works locally too). Or would you rather I move it to the setupTabs method?